.github/workflows/release.yml

```diff
@@ -1,65 +1,12 @@
 name: 🕊 Release
+
 on:
   push:
-    branches:
-      - release
-      - "release-*"
-      - "!release-experimental"
-      - "!release-experimental-*"
-      - "!release-manual"
-      - "!release-manual-*"
+    tags:
+      # only run on `remix` tags
+      - "remix@*"
```

Comment:
based on how branches work ("release-*"), this should work as the remix package's tag is "remix@1.6.5")